<?xml version="1.0" ?>
<launch>
    <rosparam file = "$(find control)/config/control_params.yaml" command = "load"/>
    <rosparam file = "$(find control)/config/system_params.yaml" command = "load"/>
    <param name="set_point" value="5" />
    
    <node name="system_response_plot" pkg="rqt_plot" type="rqt_plot" args="/setpoint /motor_output/output /motor_input/input"/>

    <node name="system"          pkg="control" type="system.py"          output="screen"/>
    <node name="controller_node" pkg="control" type="controller_node.py" output="screen"     launch-prefix="gnome-terminal --command"/>
    <node name="setpoint"        pkg="control" type="set_point_generator.py" output="screen" />
    <node name="signal_generator_graph" pkg="rqt_graph" type="rqt_graph" args=""/>
</launch>